### DISCUSSION TITLE
Report Level

### DISCUSSION THREAD

**Discussion by andreaeddyricci** [casact/chainladder-python#531] [General]
Has anyone done any work in Chainladder with Report Level?  From my understanding, the triangle has values at 18, 30, 42, 54, etc months, but the 18 (30, 42, etc) months is not a specific month but rather the value of each claim at 18 months old (which would be many different months based on the effective date of the claim.)

**Comment by kennethshsu** [comment]
Hi @andreaeddyricci, do you want to have Report Date as the rows? So basically `report_date` will be your `origins`? You can use anything as your `origins`, `accident_date`, `report_date`, or anything that you fancy. The package does not restrict you on what data to pass in.
